specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-Preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1189 @@
+          }
+        },
+        "relations" : {
+          "type" : "array",
+          "description" : "Relations of recognized biomedical entities in the document.",
+          "readOnly" : true,
+          "items" : {
+            "$ref" : "#/definitions/RelationRecord"
+          }
+        },
+        "statistics" : {
+          "description" : "(Optional) if showStats=true was specified in the request this field will contain information about the \r\n            document payload."
+        }
+      }
+    },
+    "RelationRecord" : {
+      "type" : "object",
+      "properties" : {
+        "target" : {
+          "type" : "string",
+          "description" : "TBD"
+        },
+        "source" : {
+          "type" : "string",
+          "description" : "TBD"
+        },
+        "relationType" : {
+          "type" : "string",
+          "description" : "TBD"
+        }
+      }
+    },
+    "EntityLinkingBatchResult" : {
+      "type" : "object",
+      "properties" : {
+        "documents" : {
+          "type" : "array",
+          "description" : "Response by document",
+          "readOnly" : true,
+          "items" : {
+            "$ref" : "#/definitions/EntityLinkingBatchResultItem"
+          }
+        },
+        "errors" : {
+          "type" : "array",
+          "description" : "Errors and Warnings by document",
+          "readOnly" : true,
+          "items" : {
+            "$ref" : "#/definitions/ErrorRecord"
+          }
+        },
+        "statistics" : {
+          "description" : "/// <summary>\r\n            (Optional) if showStats=true was specified in the request this field will contain information about the \r\n            request payload.\r\n            </summary>",
+          "readOnly" : true
+        }
+      }
+    },
+    "EntityLinkingBatchResultItem" : {
+      "type" : "object",
+      "properties" : {
+        "id" : {
+          "type" : "string",
+          "description" : "Unique, non-empty document identifier."
+        },
+        "entities" : {
+          "type" : "array",
+          "description" : "Recognized well-known entities in the document.",
+          "readOnly" : true,
+          "items" : {
+            "$ref" : "#/definitions/EntityLinkingRecord"
+          }
+        },
+        "statistics" : {
+          "description" : "(Optional) if showStats=true was specified in the request this field will contain information about the \r\n            document payload."
+        }
+      }
+    },
+    "EntityLinkingRecord" : {
+      "type" : "object",
+      "properties" : {
+        "name" : {
+          "type" : "string",
+          "description" : "Entity Linking formal name."
+        },
+        "matches" : {
+          "type" : "array",
+          "description" : "List of instances this entity appears in the text.",
+          "items" : {
+            "$ref" : "#/definitions/MatchRecord"
+          }
+        },
+        "wikipediaLanguage" : {
```

Comment:
I think we may want to change this to be more generic to support other data sources as well. 
For example: have a `dataSourceLanguage`, `dataSourceId`, etc. and a `dataSource` with a `Wikipedia` value.